lib/datasource/pod/index.ts

```diff
@@ -34,7 +34,7 @@ async function makeRequest<T = unknown>(
   json = true
 ): Promise<T | null> {
   try {
-    const resp = await api.get(url, { json });
+    const resp = await api.getJson<T>(url, { json });
     if (resp && resp.body) {
       return resp.body;
     }
```

Comment:
```suggestion
    return await api.getJson<T>(url, { json }).body;
```

Http (got) never returns `null`